CONTRIBUTING.md

```diff
@@ -85,40 +137,77 @@ from being shipped to production.
 
 ### Default frontmatter fields
 
-Each content type (i.e. `docs`, `guides`, `resources`, etc) support different
-custom metadata fields within the yaml frontmatter. The specific fields
+Each content type (i.e. `docs`, `guides`, `resources`, etc) has the ability to
+support different custom metadata fields within the yaml frontmatter.
 
 While each content group may support additional frontmatter fields, the
 following are default supported by all content groups:
 
 - [`title`](#title)
 - [`description`](#description)
+- [`tags`](#tags)
+- [`date`](#date)
 
-#### `title`
+#### title
 
 The primary title of the individual piece of content
 
+- name: `title`
 - type: `string`
 - required: `true`
 
-#### `description`
+#### description
 
 Brief description of the content (also used in the SEO metadata)
 
+- name: `description`
+- type: `string`
+- required: `false`
+
+> This field will be used when a short description is needed to be displayed,
+> like in typical blog style "overview cards" and for SEO related metadata.
+
+#### `tags`
+
+List of filterable tags for content
+
+- name: `tags`
+- type: `string`
+- required: `false`
+
+#### date
+
+The date this content was published.
+
+- name: `date`
 - type: `string`
 - required: `false`
 
+#### updatedDate
+
+The date this content was last updated.
+
+- name: `updatedDate`
+- type: `date`
```

Comment:
date is type string but updatedData is type 'date' is that correct? Whats the difference?